cmd/vela-worker/run.go

```diff
@@ -80,6 +83,8 @@ func run(c *cli.Context) error {
 				Limit:   c.Int("build.limit"),
 				Timeout: c.Duration("build.timeout"),
 			},
+			// build configuration
+			CheckIn: &checkIn,
```

Comment:
If we make `CheckIn` not be a pointer, then we can modify this line:

```suggestion
			CheckIn: c.Duration("checkIn"),
```